roles/servicetelemetry/templates/manifest_grafana_ds_prometheus.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: {{ ansible_operator_meta.name }}-ds-stf-prometheus
+  namespace: {{ ansible_operator_meta.namespace }}
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: "stf"
```

Comment:
In theory we could also make this dynamic and have the dashboards aligned to the ServiceTelemetry name (`default`) but we don't really have an expectation (or testing, or documentation) around running multiple STF instances in the same namespace, although historically it has seemed to mostly work. (I'd bet there are at least a few gaps now though that doesn't permit that to happen.)